app/assets/javascripts/modules/external_users/claims/FeeCalculator.js

```diff
@@ -108,12 +133,14 @@ moj.Modules.FeeCalculator = {
     .done(function(response) {
       self.clearErrors(context);
       self.setRate(response.data.amount, context);
+      self.setHint(response.data.unit, context);
```

Comment:
**[not a blocker]** be good to get this working on fixed fees too

Reply:
Added as ticket CBO-403 as this ticket only covers misc fees and I hate scope creep!